cmd/relay/relay.go

```diff
@@ -169,3 +155,132 @@ func Run(ctx context.Context, config Config) error {
 		}
 	}
 }
+
+// newENRHandler returns a handler that returns the nodes ID and public address encoded as a ENR.
+func newENRHandler(ctx context.Context, tcpNode host.Host, p2pKey *ecdsa.PrivateKey, config p2p.Config) func(ctx context.Context) ([]byte, error) {
+	// Resolve external hostname peridodically.
```

Comment:
```suggestion
	// Resolve external hostname peridodically (every 5 mins).
```

Reply:
I prefer not to put constants in comments, then stuff gets out of sync faster